core/primitives/src/views.rs

```diff
@@ -1020,8 +1020,11 @@ pub struct GasPriceView {
 /// Different types of finality.
 #[derive(Serialize, Deserialize, BorshSerialize, BorshDeserialize, Clone, Debug, PartialEq, Eq)]
 pub enum Finality {
+    #[serde(rename(serialize = "optimistic", deserialize = "optimistic"))]
```

Comment:
Why do we duplicate things? I expect `#[serde(rename = "name")]` work just fine...

Reply:
Oh I didn't know that.